slides/bitcoin-primer.tex

```diff
@@ -0,0 +1,216 @@
+		%\item<2->{completely decentral}
+		\end{itemize}
+		}
+\vspace{1em}
+\uncover<3->{Bitcoin protocol
+        \begin{itemize}
+        \item<3->{defines manner in which communication has to take place (ruleset)}
+        \end{itemize}
+        }     
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Transaction Capacity}  %example banking vs. decentral system?
+	
+	\begin{figure}[h!]
+		\center
+		\input{../assets/figures/transaction-capacity.tex}
+	\end{figure}\vspace{1em}
+
+\uncover<2->{       
+	Peer-to-Peer Network: 
+        \begin{itemize}
+		\item<1-> permissionless
+		\item<1-> censorship-resistant
+		\item<1-> No special privileges
+		\end{itemize}
+		}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Transaction Legitimacy}
+\textbf{Goal:} Ensure that a transaction was initiated by the actual owner.\\ \vspace{1em}
+
+	\begin{figure}[h!]
+		\center
+		\input{../assets/figures/transaction-encryption.tex}
+		\caption{Encryption and decryption of the transaction message.}
+		\label{fig:asymmeinfach}
+	\end{figure}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Transaction Legitimacy}
+	\vspace{1em}
+This approach ensures transaction \color{focus}authenticity \color{black}and \color{focus}integrity\color{black}.
+	\vspace{1em}
+\uncover<1->{
+	\begin{figure}[h!]
+		\center
+		\input{../assets/figures/transaction-manipulation.tex}
+	\end{figure}
+	}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Transaction Consensus}
+\textbf{Goal: }Deciding which (legitimate) transactions are valid. \\
+\vspace{1em}
+Double spend problem: \\
+\begin{figure}[h!]
+	\center
+	\input{../assets/figures/double-spend.tex}
+\end{figure}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Blocks and the Blockchain}
+Transactions are bundled into blocks \\
+\includegraphics[width=6cm]{../assets/images/block_1.png} \\
+\uncover<2->{
+Blocks are sequentially linked $\rightarrow $ blockchain \\
+\includegraphics[width=6cm]{../assets/images/blocks_3.png}
+} 
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Simplified Structure of the Blockchain}
+\begin{figure}[h!]
+	\center
+	\input{../assets/figures/blockchain-example.tex}
+\end{figure}
+\begin{itemize}
+\item{Change in a block component results in a change in the identification number.}
+\item{If an earlier block content is changed, this leads to inconsistency in the chain structure.}
+\end{itemize}
+\end{frame}
+%%%
+
+
+%%%
+\begin{frame}{Bitcoin Miner}
+Network participants, who decide to allocate computing power in order to create new blocks are called Bitcoin Miners.
+\center
+\includegraphics[width=6cm]{../assets/images/miner.png}
+
+\begin{itemize}
+\item{Miner chooses the status quo on which to base its block (most recent version of Blockchain)}
+\item{Miner chooses a subset of the transactions from its queue (mempool) to create a block, as long as they...}
+	\begin{itemize}
+	\item{\dots are legitimate}
+	\item{\dots do not conflict with other transactions}
+	\item{\dots do not exceed block size limitation}
+	\end{itemize}
+\end{itemize}
+\end{frame}
```

Comment:
I would reduce the text on this slide, but I'm not sure how this can easily be done. Maybe the figure and one statement are sufficient.